packages/assets-controllers/src/MultichainAssetsRatesController/MultichainAssetsRatesController.ts

```diff
@@ -428,8 +507,8 @@ export class MultichainAssetsRatesController extends StaticIntervalPollingContro
   }: {
     snapId: SnapId;
     handler: HandlerType;
-    params: OnAssetsConversionArguments;
-  }): Promise<OnAssetsConversionResponse> {
+    params: OnAssetsConversionArguments | OnAssetHistoricalPriceArguments;
+  }): Promise<OnAssetsConversionResponse | OnAssetHistoricalPriceResponse> {
```

Comment:
I think we can improve the types here, but it does the job so we can leave as is for now.